lib/travis/listener.rb

```diff
@@ -15,7 +15,7 @@ def config
 
   module Listener
     class Config < Travis::Config
-      define  redis:   { url: 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
+      define  redis:   { url: ENV['LISTENER_REDIS_URL'] || 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
```

Comment:
In fact, I would even be in favour of renaming the config key to `listener_redis`.